test/blackbox-tests/test-cases/github7034.t/run.t

```diff
@@ -98,12 +98,7 @@ But when lang dune is 3.3 or higher the warning becomes an error:
   Leaving directory 'outer'
   $ dune build --root=outer
   Entering directory 'outer'
-  File "vendored/inner/inner.ml", line 6, characters 11-18:
-  6 | type t = { x : int }
-                 ^^^^^^^
-  Error (warning 69 [unused-field]): unused record field x.
   Leaving directory 'outer'
-  [1]
 
 This is unexpected as vendored projects should be built according to their
```

Comment:
The comments here should be updated since they don't reflect what the test is doing now

Reply:
I changed the comment to have a header saying "notes from before".